.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,8 @@ Before creating a pull request, please file an issue in the ASF Jira system for
 
 * https://issues.apache.org/jira/projects/SOLR
 
-You will need to create an account in Jira in order to create an issue.
+For something minor (i.e. that wouldn't even be worth putting in release notes), you can skip JIRA. 
```

Comment:
```suggestion
For something minor (i.e. that wouldn't be worth putting in release notes), you can skip JIRA. 
```

I think the only issue was "even" in there.  Removing that I think is clearer (is more helpful than "minor") and doesn't display my attitude to frivolous JIRA issues :-)